.github/workflows/tests.yml

```
@@ -31,6 +31,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install setuptools wheel
+        if [ ${TORCH} == 1.13.1 ]; then pip install numpy==1.26.4; fi  # older torch versions fail with numpy 2
```

```suggestion
        if [ ${TORCH} = "1.13.1" ]; then pip install numpy==1.*; fi  # older torch versions fail with numpy 2
```

https://stackoverflow.com/questions/2237080/how-to-compare-strings-in-bash